### PR TITLE
trace-cmd: update trace-cmd and add necessary libraries

### PR DESCRIPTION
**Pull request by PolynomialDivision**
Update trace-cmd and add necessary libraries. Trace-cmd was split up into trace-cmd, libtraceevent and libtracefs.

### CODE REVIEW

File: package/devel/trace-cmd/Makefile [hunk 1]
```diff
@@ -44,19 +44,6 @@ TARGET_CFLAGS += --std=gnu99 -D_GNU_SOURCE
 define Package/trace-cmd/install
 	$(INSTALL_DIR) $(1)/usr/bin $(1)/usr/lib/traceevent/plugins
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/trace-cmd $(1)/usr/bin/
-	$(CP) \
-		$(patsubst %,$(PLUGINS_DIR)/plugin_%.so,$(PLUGINS_MAIN)) \
```
Comment (hauke) [inline package/devel/trace-cmd/Makefile:48]: Please remove the definition of `PLUGINS_DIR` and `PLUGINS_MAIN` from this package Makefile, they are not used any more.

File: package/libs/libtraceevent/Makefile [hunk 2]
```diff
@@ -1,12 +1,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libtraceevent
-PKG_VERSION:=1.6.1
+PKG_VERSION:=1.6.2
```
Comment (hauke) [inline package/libs/libtraceevent/Makefile:4]: Please squash the version updates into the initial versions. As you are anyway adding these as new libraries or jumping over many versions no need for separate commits.

**Comment by PolynomialDivision** [comment]
@hauke Done. :)

**Comment by hauke** [comment]
Thank you for your patches, I applied them to master in b108810601ff1976a0c33f9c20186e50f5a74e7d, 6d84e78cd9633578d33a46b914a9cc5bae082f6b and 9edda3169f6ad8f2f20b6e540ecdced40d530585.

**Pull request closed without merging**